src/streams/send.rs

```diff
@@ -1016,6 +1016,37 @@ impl StreamTx {
         self.padding = 0;
     }
 
+    /// Reset this stream to use a new SSRC.
+    ///
+    /// This updates the SSRC and resets all relevant internal fields.
+    pub(crate) fn reset_ssrc(&mut self, new_ssrc: Ssrc) {
+        // Update the SSRC
+        self.ssrc = new_ssrc;
```

Comment:
I'm wondering if we should reset the RTX SSRC to, you could imagine the remote associating the RTX SSRC with the main SSRC such that any future sends on the RTX end up going to the old one

Reply:
Yeah, makes sense. Did you observe that with FF?

Reply:
I'll check

Reply:
![image](https://github.com/user-attachments/assets/9c8444fb-c36d-46c7-a827-1efc81908bb4)

I did get new RTX streams (look at the Payload, SSRC, and start time)

Reply:
👍 Super! Thanks!